docs/other/install-gwms-frontend.md

```diff
@@ -441,11 +441,98 @@ Proxy Configuration
 
 There are 2 types of (or purposes for) proxies required for the VO Frontend: 1 the %GREEN%VO Frontend proxy%ENDCOLOR% (used to authenticate with the other glideinWMS services) 1 one or more glideinWMS %GREEN%pilot proxies%ENDCOLOR% (used/delegated to the factory services and submitted on the glideinWMS pilot jobs) The %GREEN%VO Frontend proxy%ENDCOLOR% and the %GREEN%pilot proxy%ENDCOLOR% can be the same. By default, the VO Frontend will run as user `frontend` (UID is machine dependent) so these proxies must be owned by the user `frontend`.
 
-### Manual proxy renewal
+
+- [Automatic renewal without VOMS Admin](#automatic-renewal-without-voms-admin) **(Recommended)**
+- [Manual renewal](#manual-proxy-renewal)
+- [Example of automatic renewal](#example-of-automatic-proxy-renewal)
+
+### Automatic Renewal without VOMS Admin
+
+This section describes how to automatically renew your frontend and pilot proxies without a VOMS Admin server, which is now [deprecated](https://opensciencegrid.github.io/technology/policy/voms-admin-retire/) in the OSG. If your VO uses non-OSG VOMS Admin servers, consult [this section](#example-of-automatic-proxy-renewal) instead
+
+!!! note
+    You must have access to the certificate/key pair corresponding to your VO's entry in `/etc/vomses` of the `vo-client` package. If you do not have access to these credentials, contact your [VO manager](http://oim.opensciencegrid.org/oim/vo).
+
+#### Obtaining information from your current proxy
+
+The first step is to obtain information from your current proxy. Assuming the location of your current proxy is at `<PATH_TO_OLD_PROXY>`. Then run the following command to obtain information from your current proxy:
+
+    :::console
+    root@host # voms-proxy-info --file %RED%<PATH_TO_OLD_PROXY>%ENDCOLOR% -all
+    subject   : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu/CN=287414457
+    issuer    : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    identity  : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    type      : RFC compliant proxy
+    strength  : 1024 bits
+    path      : credential_chtc2.main_745540
+    timeleft  : 5441:28:37
+    key usage : Digital Signature, Key Encipherment, Data Encipherment
+    === VO GLOW extension information ===
+    VO        : GLOW
+    subject   : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=pilot/glidein2.chtc.wisc.edu
+    issuer    : /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=glow-voms.cs.wisc.edu
+    %BLUE%attribute : /GLOW/Role=htpc/Capability=NULL %ENDCOLOR%
+    timeleft  : 71:34:02
+    %BLUE%uri       : glow-voms.cs.wisc.edu:15001 %ENDCOLOR%
+
+The most important fields to take notice are `attribute`, `VO` and `uri`. Write them down since they would be needed in the next steps
```

Comment:
- `VO`: this should either be self-evident or they should be able to get it from OIM, no?
- `uri`: once they know the VO, they can get this from the VO `.lsc` 
- `attribute`: this is the one that's troubling to me, not sure what to tell users here for new installs. @bbockelm, thoughts?